.profile.d/heroku-metrics-daemon.sh

```diff
@@ -37,7 +37,7 @@ if [[ -f build.sbt ]]; then
     unzip -qq bin/heroku-metrics-agent.jar 'javax/*' -d bin/ext/
     export JAVA_OPTS="-javaagent:bin/heroku-metrics-agent.jar=cp=/app/bin/ext/ -Xbootclasspath/a:bin/heroku-metrics-agent.jar ${JAVA_OPTS}"
     AGENTMON_FLAGS+=("-prom-url=http://localhost:${HEROKU_METRICS_PROM_PORT}${HEROKU_METRICS_PROM_ENDPOINT}")
-elif [[ -f pom.xml ]] || [[ -f build.gradle ]] || [[ -d .jdk ]]; then
+elif [[ -f pom.xml ]] || [[ -f build.gradle ]] || [[ -f project.clj ]] || [[ -d .jdk ]]; then
```

Comment:
It would look like:

```
detect_jvm_app() {
  local app_dir=${1:-'.'}
  test -f "${app_dir}/pom.xml" ||
    test -f "${app_dir}/build.gradle" ||
    test -f "${app_dir}/build.sbt" ||
    test -f "${app_dir}/project.clj" ||
    test -d "${app_dir}/.jdk"
}
```